src/components/Kv/KvPillToggle.vue

```diff
@@ -17,17 +26,34 @@ export default {
 	props: {
 		options: {
 			type: Array,
-			required: true
+			required: true,
+			default() {
+				return [
+					{
+						title: '',
+						key: '',
+						disabled: false
+					}
+				];
+			}
 		},
 		selected: {
 			type: String,
-			required: true
+			default: '',
 		},
 	},
+	data() {
+		return {
+			checked: ''
```

Comment:
Alternatively you can just bind checked to selected here and avoid the created hook all together...
```suggestion
			checked: this.selected
```

Reply:
Nice, thanks!